commons/ava_testsuite/five_node_staking_gecko_network_tests.go

```diff
@@ -0,0 +1,73 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_networks"
+	"github.com/kurtosis-tech/ava-e2e-tests/gecko_client"
+	"github.com/kurtosis-tech/kurtosis/commons/testsuite"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+
+type FiveNodeStakingNetworkBasicTest struct{}
+func (test FiveNodeStakingNetworkBasicTest) Run(network interface{}, context testsuite.TestContext) {
+	castedNetwork := network.(ava_networks.NNodeGeckoNetwork)
+
+	// TODO check ALL nodes!
+	client, err := castedNetwork.GetGeckoClient(0)
+	if err != nil {
+	context.Fatal(stacktrace.Propagate(err, "Could not get client"))
```

Comment:
What's going on with the indendentation here

Reply:
done